cl/recap/models.py

```diff
@@ -211,6 +212,53 @@ def file_contents(self):
     def print_file_contents(self):
         print(self.file_contents)
 
+class EmailProcessingQueue(AbstractDateTimeModel):
+    uploader = models.ForeignKey(
+        User,
+        help_text="The user that uploaded the item to RECAP.",
+        related_name="recap_email_processing_queue",
+        on_delete=models.CASCADE,
+        null=True,
+    )
+    court = models.ForeignKey(
+        Court,
+        help_text="The court where the upload was from",
+        related_name="recap_email_processing_queue",
+        on_delete=models.CASCADE,
+        null=True,
+    )
+    filepath = models.FileField(
+        help_text="The filepath to the disclosure normalized to a PDF.",
+        upload_to=make_recap_email_processing_queue_aws_path,
+        storage=AWSMediaStorage(),
+        max_length=300,
+        null=True,
+    )
+    status = models.SmallIntegerField(
+        help_text="The current status of this upload. Possible values "
+                  "are: %s"
+                  % ", ".join(
+            ["(%s): %s" % (t[0], t[1]) for t in PROCESSING_STATUS.NAMES]
+        ),
+        default=PROCESSING_STATUS.ENQUEUED,
+        choices=PROCESSING_STATUS.NAMES,
+        db_index=True,
+    )
+    error_message = models.TextField(
+        help_text="Any errors that occurred while processing an item",
+        blank=True,
+    )
+    recap_documents = models.ManyToManyField(
+        RECAPDocument,
+        related_name="recap_email_processing_queue",
+        help_text="The document that was created or updated by this request, "
+                  "if applicable. Only applies to PDFs uploads.",
+        null=True
+    )
```

Comment:
Looks like you didn't include the `docket` or `docket_entries` FKs. Seems right to exclude them. You need to pluralize `document` though in your help_text. Otherwise, makes sense to me so far.

Reply:
Oh, well, at least I'm pretty consistent.